client/src/components/Icons.jsx

```diff
@@ -0,0 +1,258 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Search from "./Search.jsx";
+
+class Icons extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			search: false
+		};
+
+		this.closeNav = this.closeNav.bind(this);
+		this.openNav = this.openNav.bind(this);
+		this.openSearch = this.openSearch.bind(this);
+	}
+
+	//onClick function for the menu bars
+	closeNav() {
+		document.getElementById("sideNav").style.width = "0";
+	}
+
+	openNav() {
+		document.getElementById("sideNav").style.width = "250px";
+	}
+
+	openSearch() {
+		this.setState(prevState => ({
+			search: !prevState.search
+		}));
+	}
+
+	render() {
+		return (
+			<div>
+				<div id="sideNav">
+					<a
+						id="closeBtn"
+						onClick={() => {
+							this.closeNav();
+						}}
+					>
+						{" "}
+						&times;
+					</a>
+					<a href="https://www.hostelworld.com/"> 
+						<i class="fas fa-home fa-sm"></i> 
+						Home 
+					</a>
+					<br />
+					<a data-toggle="modal" data-target="#exampleModal"> 
+						<i class="fas fa-sign-in-alt fa-sm"></i>
+						Sign In/ Create Account 
+					</a>
+					<br />
+					<p> Language 
+						<select>
```

Comment:
suggesting you making an array of languages such as:
`var languages  = ['English', 'Espanol', Italiano']'`
then render option according to each
`languages.forEach((lang, index) => return <option key={index}>{lang}</option>`
Same thing can be done for a lot of parts below that you hardcoded.